deepmd/pt/model/model/dp_zbl_model.py

```diff
@@ -120,3 +127,27 @@ def update_sel(cls, global_jdata: dict, local_jdata: dict):
             global_jdata, local_jdata["dpmodel"]
         )
         return local_jdata_cpy
+
+    @classmethod
+    def deserialize(cls, data) -> "DPZBLModel":
```

Comment:
the method is simply a copy of the atomic dp zbl model.

Reply:
I have no idea how to convert a DP AtomicZBLModel to a DPZBLModel